helmfile/charts/starboard/ciskubebench-exporter/templates/rbac.yaml

```diff
@@ -0,0 +1,35 @@
+---
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: ciskubebench-exporter
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: ciskubebench-exporter
+rules:
+  - apiGroups: ["aquasecurity.github.io"]
+    resources: ["ciskubebenchreports"]
+    verbs: ["get", "list", "watch"]
+  - apiGroups:
+    - extensions
+    resourceNames:
+    - privileged
+    resources:
+    - podsecuritypolicies
+    verbs:
+    - use
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
```

Comment:
Couldn't this be a role and a rolebinding instead of clusterrole and clusterrolebinding?

Reply:
^